pkg/resources/schema_acceptance_test.go

```diff
@@ -501,6 +502,28 @@ func TestAcc_Schema_ManagePublic(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_schema.test", "pipe_execution_paused", "false"),
 				),
 			},
+			// make sure the schema wasn't dropped
```

Comment:
I think the steps before this one refer to the 0.94.0 version

Reply:
Specified 0.94 for the steps above.